atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/table/RegeneratingTable.java

```diff
@@ -61,15 +63,23 @@ public TableReference getTableRef() {
         return Tables.TABLE_REF;
     }
 
+    public SweepTaskRunner getSweepTaskRunner() {
+        return services.getSweepTaskRunner();
+    }
+
     @Setup(Level.Trial)
     public void setup(AtlasDbServicesConnector conn) {
         this.connector = conn;
         this.services = conn.connect();
+        setupTable();
+        setupTableData();
+    }
+
+    protected void setupTable() {
```

Comment:
Perhaps we want (1) this to return a table reference, (2) setup() to store this table reference, and (3) getTableRef() to return that table reference?  This is not a big deal, but may prevent people from accidentally not using the correct table reference?  Alternatively, make setupTable() be createTable(TableReference) so that the implementer knows they have to create that table ref.  I think I like the second one better.  Regardless, this is a nit -- I'm happy to merge this.